docs/content/preview/explore/query-1-performance/pg-stat-progress-create-index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: View CREATE INDEX status with pg_stat_progress_create_index
+linkTitle: View CREATE INDEX status
+description: Use pg_stat_progress_create_index to get the CREATE INDEX command status, including the status of an ongoing concurrent index backfill, and the index build's progress reports.
+headerTitle: View CREATE INDEX status with pg_stat_progress_create_index
+image: /images/section_icons/index/develop.png
+menu:
+  preview:
+    identifier: pg-stat-progress-create-index
+    parent: query-tuning
+    weight: 450
+type: docs
+---
+
+You can add a new index to an existing table using the YSQL [CREATE INDEX](../../../api/ysql/the-sql-language/statements/ddl_create_index/#semantics) statement. YugabyteDB supports [online index backfill](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/online-index-backfill.md), which is enabled by default. Using this feature, you can build indexes on non-empty tables online, without failing other concurrent writes. YugabyteDB also supports the [CREATE INDEX NONCONCURRENTLY](../../../api/ysql/the-sql-language/statements/ddl_create_index/#nonconcurrently) statement to disable online index backfill.
+
+### pg_stat_progress_create_index
+
+YugabyteDB supports the PostgreSQL `pg_stat_progress_create_index` view to report the progress of the CREATE INDEX command execution. Whenever CREATE INDEX is running, the `pg_stat_progress_create_index` view contains one row for each client connection that is currently running a CREATE INDEX command, and the row entry is cleared after the completion of command execution.
```

Comment:
for each backend instead of for each client connection to be precise

Reply:
also i see that this is from the postgresql docs so you can skip this suggestion but - 

instead of "Whenever CREATE INDEX is running, the `pg_stat_progress_create_index` view contains one row for each client connection that is currently running a CREATE INDEX command, ..." (which refers to a running CREATE INDEX twice)

I think 

"The view contains one row for each backend that is currently running a CREATE INDEX command, and the row entry is cleared after the completion of command execution." 
would be better

Reply:
Thanks @fizaaluthra . Yes Ill make the changes.